doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -533,15 +571,73 @@ To use full QuantumGraph file, the submit YAML must set `whenSaveJobQgraph` to
     runQuantumCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} --output {output} --output-run {outCollection} --qgraph {qgraphFile} --qgraph-id {qgraphId} --qgraph-node-id {qgraphNodeId} --skip-init-writes --extend-run --clobber-outputs --skip-existing"
 
 
--- warning::
+.. warning::
 
    Do not modify the QuantumGraph options in pipetaskInit's runQuantumCommand.  It needs the entire QuantumGraph.
 
--- note::
+.. note::
 
    If running on a system with a shared filesystem, you'll more than likely want to also set bpsUseShared
    to true.
 
+.. _execution-butler:
+
+Execution Butler
+----------------
+
+Execution butler is a behind-the-scenes mechanism to lessen the number
+of simultaneous connections to the Butler database.
+
+
+.. _DMTN-177: https://github.com/lsst-dm/dmtn-177
+
+
+Pipetask command lines are not the same when using execution butler.
+There is currently not a single configuration option to enable/disable
+execution butler.
+
+Command-line Changes
+^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: YAML
+
+   pipetask:
+       pipetaskInit:
+           runQuantumCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output-run {outCollection} --init-only --register-dataset-types --qgraph {qgraphFile} --extend-run {extraInitOptions}"
+   runQuantumCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} --output-run {outCollection} --qgraph {qgraphFile} --qgraph-id {qgraphId} --qgraph-node-id {qgraphNodeId} --skip-init-writes --extend-run --clobber-outputs --skip-existing {extraRunQuantumOptions}"
+
+New section
+^^^^^^^^^^^
+
+.. code-block:: YAML
+
+   executionButler:
+       whenCreate: "SUBMIT"
+       #USER executionButlerDir: "/my/exec/butler/dir"  # if user provided, otherwise uses executionButlerTemplate
+       createCommand: "${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -b {butlerConfig} --input {inCollection} --output-run {outCollection} --save-execution-butler {executionButlerDir} -g {qgraphFile}"
+       whenMerge: "ALWAYS"
+       implementation: JOB  # JOB, WORKFLOW
+       concurrency_limit: db_limit
+       command1: "${DAF_BUTLER_DIR}/bin/butler --log-level=VERBOSE transfer-datasets  {executionButlerDir} {butlerConfig} --collections {outCollection}"
+       command2: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} {outCollection} --mode=prepend"
+
+For --replace-run behavior need to replace the one collection-chain command with these two:
+
+.. code-block:: YAML
+
+   command2: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} --mode=pop 1"
+   command3: "${DAF_BUTLER_DIR}/bin/butler collection-chain {butlerConfig} {output} --mode=prepend {outCollection}"
+
+**whenCreate**
+    When during the submission process that the execution butler is created.
+    whenCrete valid values: "NEVER", "ACQUIRE", "TRANSFORM", "PREPARE",
+"SUBMIT"
```

Comment:
With the current formatting, Sphinx will treat `"SUMBIT"` as a new entry in the description and I suspect (judging from the text itself) that's not the case.

Reply:
This file was updated since this comment was made.